docs/plugins/clean-url-tracker.md

```diff
@@ -0,0 +1,127 @@
+# `cleanUrlTracker`
+
+This guide explains what the `cleanUrlTracker` plugin is and how to integrate it into your `analytics.js` tracking implementation.
+
+## Overview
+
+When viewing your most visited pages in Google Analytics, it's not uncommon to see multiple different URL paths that reference the same page on your site. The following table is a good example of this and the frustrating situation many users find themselves in today:
+
+<table>
+  <tr valign="top">
+    <th align="left">Page</th>
+    <th align="left">Pageviews</th>
+  </tr>
+  <tr valign="top">
+    <td>/contact</td>
+    <td>967</td>
+  </tr>
+  <tr valign="top">
+    <td>/contact/</td>
+    <td>431</td>
+  </tr>
+  <tr valign="top">
+    <td>/contact?hl=en</td>
+    <td>67</td>
+  </tr>
+  <tr valign="top">
+    <td>/contact/index.html</td>
+    <td>32</td>
+  </tr>
+</table>
+
+To prevent this problem, it's best to settle on a single, canonical URL path for each page you want to track, and only ever send the canonical version to Google Analytics.
+
+The `cleanUrlTracker` plugin helps you do this. It lets you specify a preference for whether or not to include extraneous parts of the URL path, and updates all URLs accordingly.
+
+### How it works
+
+The `cleanUrlPlugin` works by intercepting each hit as it's being sent and modifying the [`page`](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#page) field based on the rules specified by the configuration [options](#options).
+
+If no `page` field exists, one is created based on the URL path from the [`location`](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#location) field. Note that while the `cleanUrlTracker` plugin does modify `page` field values for each hit, it never modifies the `location` field. This allows campaign and site search data encoded in the full URL to be preserved.
```

Comment:
I would make the Note a separate paragraph, potentially bold Note: to highlight.